.happy/terraform/envs/prod/outputs.tf

```
@@ -0,0 +1,19 @@
+output frontend_url {
+  value       = module.stack.frontend_url
+  description = "The URL endpoint for the frontend website service"
+}
+
+output backend_url {
+  value       = module.stack.backend_url
+  description = "The URL endpoint for the backend website service"
+}
+
+output delete_db_task_definition_arn {
+  value       = module.stack.delete_db_task_definition_arn
+  description = "ARN of the ECS Task Definition that deletes dev databases"
```

do we need this if the db is deletion protected? if so can we change it to the following:
```suggestion
  description = "ARN of the ECS Task Definition that deletes prod database"
```